webpack.common.js

```diff
@@ -39,7 +39,7 @@ function configure(IS_TEST) {
         moduleExtensions: ['-loader']
       },
     resolve: {
-      extensions: ['.json', '.js', '.jsx', '.ts', '.tsx', '.css', '.less', '.html'],
+      extensions: ['.json', '.ts', '.tsx', '.js', '.jsx', '.css', '.less', '.html'],
```

Comment:
why

Reply:
If there are two files in the directory with the same extension, it will load them in order. Generally .ts and .tsx are the newer ones, so guarding against old files hanging around breaking angular loading at runtime.